src/components/11-button-groups/button-groups--segmented.njk

```diff
@@ -0,0 +1,64 @@
+<h6>Segmented buttons</h6>
+<ul class="usa-button-group usa-button-group--segmented">
+  <li class="usa-button-group__item">
+    <button class="usa-button">Button</button>
+  </li>
+  <li class="usa-button-group__item">
+    <button class="usa-button">Button</button>
+  </li>
+  <li class="usa-button-group__item">
+    <button class="usa-button">Button</button>
+  </li>
+</ul>
+
+<h6>Segmented outline buttons</h6>
+<ul class="usa-button-group usa-button-group--segmented">
+  <li class="usa-button-group__item">
+    <button class="usa-button usa-button--outline">Button</button>
+  </li>
+  <li class="usa-button-group__item">
+    <button class="usa-button usa-button--outline">Button</button>
+  </li>
+  <li class="usa-button-group__item">
+    <button class="usa-button usa-button--outline">Button</button>
+  </li>
+</ul>
+
+<h6>Segmented secondary buttons</h6>
+<ul class="usa-button-group usa-button-group--segmented">
+  <li class="usa-button-group__item">
+    <button class="usa-button usa-button--secondary">Button</button>
+  </li>
+  <li class="usa-button-group__item">
+    <button class="usa-button usa-button--secondary">Button</button>
+  </li>
+  <li class="usa-button-group__item">
+    <button class="usa-button usa-button--secondary">Button</button>
+  </li>
+</ul>
```

Comment:
`secondary` buttons are typically reserved for destructive actions, but I'm not sure if I've seen destructive actions in a segmented group before. The CSS supports this, but do we want to showcase this example?

Reply:
Maybe not with a destructive action, but could we show an example with an "active/current" button — that is, is there a button state that communicates "there are three options, and currently you've selected the third of them"?

Reply:
@thisisdano This is an outdated diff. Here's what I'm showing after [this commit](https://github.com/uswds/uswds/pull/3374/commits/0620bf3cfae19a66b16d421231e9af906034566b) now:
https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.app.cloud.gov/preview/uswds/uswds/10x-forms/button-groups/components/detail/button-groups--segmented.html

Reply:
grrr. sorry about that

Reply:
Poifect